plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StandardFacetInstallDelegate.java

```diff
@@ -75,6 +75,7 @@ private AppEngineRuntimeInstallJob(IFacetedProject facetedProject) {
       super(Messages.getString("appengine.install.runtime.to.project", // $NON-NLS$
           facetedProject.getProject().getName()));
       this.facetedProject = facetedProject;
+      setRule(facetedProject.getProject());
```

Comment:
I see what you mean 😭  I think the `ConvertJob` will begin — but it may become 'paused' in the middle when it tries to `commitChanges()` and acquire a workspace lock.  That said, the `NonSystemJobSuspender` will suspend this job, won't it?